### PR TITLE
Add applicationinsights sample

### PR DESCRIPTION
**Pull request by ychikazawa**
Add telegraf + application insights sample

### CODE REVIEW

File: windows_metrics_telegraf_applicationinsights/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+Instead, creating an Application Insights resource.
+
+## Usage
+
+You should download telegraf from the [GitHub releases](https://github.com/influxdata/telegraf/releases) section. The [Windows amd64 package](https://dl.influxdata.com/telegraf/releases/telegraf-1.24.4_windows_amd64.zip) works with this sample.
+Then, you should create a [Application Insights](https://learn.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview) resource in your subscription.
+Copy the Instrumentation key from the Overview of the Application Insights resource you created and paste it into the telegraf.conf file.
+
+
+We need telegraf to be able to authenticate to Application Insights on your subscription so it can post the metrics. Use the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/) to create a service principal with the "Monitoring Metrics Publisher" role for the subscription that metrics will be published against (subscription ID can be retrieved using `az account list` or from the Azure portal):
```
Comment (dgkanatsios) [inline windows_metrics_telegraf_applicationinsights/README.md:28]: Do we need to create a Service Principal in this case? Telegraf would use the AppInsights Instrumentation Key to authenticate, no?

File: windows_metrics_telegraf_applicationinsights/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+You can see documentation about Application Insights output telegraf plugin [here](https://github.com/influxdata/telegraf/blob/master/plugins/outputs/application_insights/README.md).
+
+## What it does
+
+This script uses [telegraf](https://www.influxdata.com/time-series-platform/telegraf/) agent to send VM metrics to Application Insights. The telegraf agent publishes [custom metrics](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-custom-overview) using the Azure Monitor REST API. Custom metrics are available in these [regions](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-custom-overview#supported-regions).
+
+This script is applicable if you are running Windows MPS Builds using Windows containers or processes for your game servers.
+
+The result is almost the same as [Windows metrics with telegraf and Azure Monitor](../windows_metrics_telegraf_azuremonitor/README.md).
+But There is no need to prepare a dummy VMSS.
```
Comment (dgkanatsios) [inline windows_metrics_telegraf_applicationinsights/README.md:18]: nit: "There" should be "there"

File: windows_metrics_telegraf_applicationinsights/PF_StartupScript.ps1 [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+((Get-Content -path $telegrafConfPath -Raw) -replace '_%PF_VM_ID%_', "$env:PF_VM_ID") | Set-Content -Path $telegrafConfPath
+
+# install telegraf as a Windows service
+$telegrafPath = 'C:\Program Files\telegraf'
+New-Item -ItemType Directory -Force -Path "$telegrafPath"
+cp "$scriptPath\telegraf.*" "$telegrafPath"
+cd "$telegrafPath"
+.\telegraf.exe --service install --config "$telegrafPath\telegraf.conf"
+
+# configure environment variables for the telegraf service
```
Comment (dgkanatsios) [inline windows_metrics_telegraf_applicationinsights/PF_StartupScript.ps1:39]: similar to the other comment, do we need these values?

File: windows_metrics_telegraf_applicationinsights/README.md (hunk 2 above)
Comment (ychikazawa) [reply windows_metrics_telegraf_applicationinsights/README.md]: I fixed it!

File: windows_metrics_telegraf_applicationinsights/README.md (hunk 1 above)
Comment (ychikazawa) [reply windows_metrics_telegraf_applicationinsights/README.md]: You're right. we don't need this. I removed!

File: windows_metrics_telegraf_applicationinsights/PF_StartupScript.ps1 (hunk 3 above)
Comment (ychikazawa) [reply windows_metrics_telegraf_applicationinsights/PF_StartupScript.ps1]: Is this referring to the registry part? That was unnecessary and I removed it.

**Review by dgkanatsios — APPROVED** [review]
:shipit:

**Pull request merged**